### PR TITLE
layout change

### PR DESCRIPTION
**Pull request by dejanlaco**
As requested, the layout is changed here

### CODE REVIEW

File: docroot/monitor.php [hunk 1]
```diff
@@ -155,48 +155,28 @@ function ChangeCheckbox(elementId) {
...
+            <?= $info->GetMostLiked($filter); ?>
+            <hr />
+            <h3>Most Commented...</h3>
+            <?= $info->GetMostCommented($filter); ?>
+            <hr />
+            <h3>Latest post...</h3>
+            <?= $info->GetLatestPost($filter); ?>
+            <hr />
+            <h3>Latest image...</h3>
+            <img class="post_img" src=<?= $info->GetMedia($filter); ?>>
```
Comment (0xF1o) [inline docroot/monitor.php]: maybe add quotes to the src attribute...

**Comment by 0xF1o** [comment]
I did not request any layout changes, nor am i aware of any changes being requested...

**Comment by Grienauer** [comment]
the layout of the backend looks cleaner with a stacked layout. I requested it :)

**Comment by 0xF1o** [comment]
ok, then please remove me as reviewer - i cant do that myself

**Review by Grienauer — APPROVED** [review]

**Comment by Grienauer** [comment]
@piccaso looks good to me!

**Pull request merged**